contribs/src/main/java/com/netflix/conductor/contribs/queue/amqp/AMQPObservableQueue.java

```diff
@@ -63,6 +65,7 @@
     private Channel channel;
     private final Address[] addresses;
     protected LinkedBlockingQueue<Message> messages = new LinkedBlockingQueue<>();
+    private final AtomicBoolean running = new AtomicBoolean();
```

Comment:
The `AtomicBoolean` seems unnecessary here, since there is no "compare and set" operation. For this scenario, a `volatile boolean` should suffice.

Reply:
good idea, changed